### PR TITLE
edited readme and travis passed

### PR DESCRIPTION
**Pull request by meets2tarun**
#12 #1 #9

### CODE REVIEW

File: etc/system-lock/lock.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ fi
 sudo chmod +x /etc/rc.local
 fi
 
-if [ $is_success = true ]; then
-sudo reboot
-fi
+#if [ $is_success = true ]; then
+#sudo reboot
```
Comment (meets2tarun) [inline etc/system-lock/lock.sh]: @mariobehling I will clear the reboot issue in the next commit.

**Comment by meets2tarun** [comment]
@niccokunzmann @mariobehling 
Please review and merge

**Comment by meets2tarun** [comment]
Sir, is there any issue right now?

**Comment by Abhi2424shek** [comment]
Nothing IMO

File: etc/system-lock/lock.sh (hunk 1 above)
Comment (niccokunzmann) [inline etc/system-lock/lock.sh]: #16

**Comment by meets2tarun** [comment]
thanks

**Pull request merged**